app/jobs/session_trim_job.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class SchoolDataImporterJob < CronJob
```

Comment:
This is not going to do anything by itself. We need to go back to where Michal lifted this from, and get the final piece for this to work. When we do that (now?), we need to prevent the _actual_ SchoolDataImporterJob from running.

Reply:
Someone's looking at an old version ;)

Reply:
I have added `cron:schedule` rake task which re-schedules selected jobs :)